spring-boot/config-server/src/main/java/eu/coatrack/admin/server/service/filter/CheckIfUrlAndCredentialsFitWithEachOtherFilter.java

```diff
@@ -77,7 +72,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                             log.debug("Credential was found: {} -> this user is authorized to access this resource", credentialForUserAndResource);
                         } else {
                             log.debug("No Credential was found for user {} and resource {}", user, url);
-                            httpResponse.sendError(HttpServletResponse.SC_FORBIDDEN, "You do not have permission to access: " + url);
+                            servletResponse.sendError(HttpServletResponse.SC_FORBIDDEN, "You do not have permission to access: " + url);
                         }
 
                         // Retrieve the Id in Long format TODO Consider to separate the business key to the technical key which generates such kind of dirty logic
```

Comment:
Is that dead code? If not, this should imo be an issue and not clutter the code.